checkstyle.xml

```diff
@@ -82,6 +82,11 @@
         <module name="ArrayTypeStyle"/>
         <module name="MissingSwitchDefault"/>
         <module name="FallThrough"/>
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+        <module name="StringLiteralEquality"/>
+        <module name="UnnecessaryParentheses"/>
+        <module name="ExplicitInitializationCheck"/>
```

Comment:
>  So there is a minor inefficiency

Which is completely irrelevant. The code is more readable if you can go to the declaration and see what the initial value is *immediately* without having to remember or google what the default value is